README.md

```diff
@@ -1,7 +1,7 @@
 # ArchivesSpace Oauth
 
 Configure ArchivesSpace as a service provider (SP) for oauth authentication.
-*The plugin delegates authentication to the configured identity provider (IDP).*
+_The plugin delegates authentication to the configured identity provider (IDP)._
 
```

Comment:
Ooops, didn't mean to change this, my editor auto-formatted it. Happy to revert.

Reply:
No need for that. It's fine.